test_elasticsearch/test_server/test_helpers_vectorstore/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
+  elasticsearch:
+    image: elasticsearch:8.13.0
+    environment:
+      - discovery.type=single-node
+      - xpack.license.self_generated.type=trial
+      - xpack.security.enabled=false  # disable password and TLS; never do this in production!
+    ports:
+      - "9200:9200"
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "curl --silent --fail http://localhost:9200/_cluster/health || exit 1"
+        ]
+      interval: 10s
+      retries: 60
+
+  # Currently fails on Mac: https://github.com/elastic/elasticsearch/issues/106206
+  elasticsearch-with-models:
+    image: docker.elastic.co/eland/eland
+    depends_on:
+      - elasticsearch
+    restart: no
+    command: sh -c "sleep 10 && eland_import_hub_model --url http://elasticsearch:9200 --hub-model-id sentence-transformers/msmarco-minilm-l-12-v3 --start"
+
+  kibana:
```

Comment:
Do we need Kibana for anything here? Based on the work I've done to get an easy method of installation, I believe this is not going to give you a working Kibana since the Kibana password isn't configured.

Reply:
No, don't need it. Left-over from the original file I copied. Removing.